app/views/students/reviews/index.html.erb

```diff
@@ -1,13 +1,18 @@
-<h1 class="subtitle">Your reviews</h1>
-
-<div id="reviews">
+<div class="container pt-5">
+  <h1 class="title">Your reviews</h1>
   <br />
+  
   <% @reviews.each do |review| %>
     <%= render "students/reviews/review", review: review %>
     <br>
   <% end %>
-</div>
 
-<%= will_paginate @reviews, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
-<br />
-<%= link_to "Back", student_url(@user) %>
+  <%= will_paginate @reviews, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
```

Comment:
You have a mixture between `:class=>""`, `class=""` and `class: ""`
Choose one and keep a consistency between all you app

Reply:
Thanks I'll keep in mind for the next changes